packages/documentation-site/patternfly-docs/content/get-started/contribute.md

```diff
@@ -0,0 +1,69 @@
+- New feature ideas. 
+- Bug reports.
+- Documentation updates.
+
+This guide outlines the ways that you can contribute to PatternFly's design, code, and documentation. With your help, we can stay on top of the latest and greatest solutions to keep PatternFly moving forward. 
+
+If you have any ideas that don't fit into the projects outlined in this guide, please [reach out to us on the primary 'patternfly' Slack channel](https://patternfly.slack.com/archives/C293LQ36J) to start a discussion.
+
+## Design
+
+If you have skills in visual and interaction design, you can contribute to PatternFly's design, by taking an existing issue, or proposing a new design system feature, enhancement, or icon. If you are interested in any of these projects, [reach out on the 'patternfly-design' Slack channel.](http://join.slack.com/t/patternfly/shared_invite/zt-1npmqswgk-bF2R1E2rglV8jz5DNTezMQ)
+
+### Existing design issues 
+
+The PatternFly design team is composed of visual and interaction designers who define the look and feel of the PatternFly library. The team follows an agile framework, planning their work in sprints, with a backlog that is tracked and managed via [this GitHub project board.](https://github.com/orgs/patternfly/projects/7/views/30) This board contains a list issues that are currently unassigned and waiting in the queue. If you see something here that you'd like to work on, leave a comment on the issue and a member of our team will reach out with next steps.
+
+### New feature or enhancement
+If you have an idea for a new design pattern, a new component type, or have ideas for improving existing features, we'd love to hear it. [Start by opening an issue in the `patternfly-design` repository.](https://github.com/patternfly/patternfly-design/issues) From there, a member of our team will reach out and work with you to plan and design a solution.
+
+### New icons
+We encourage designers to work with [the existing PatternFly icon set](/design-foundations/icons), which includes both custom and existing Font Awesome icons that cover most common use cases. However, when a use case isn't covered, you may want to propose adding a new icon to our set.
+
+To contribute a new icon, [start by opening an issue in the `patternfly-design` repository](https://github.com/patternfly/patternfly-design/issues), proposing your icon idea and describing why it's needed. A member of our team will reach out to you to discuss next steps.
+
+## Code
+
+The primary PatternFly libraries include React and HTML/CSS (commonly called "core"). If you're looking to contribute to PatternFly's codebase, these libraries are a good place to start. You can help out by taking existing issues, or creating issues for bugs and other changes. 
```

Comment:
Here I'd switch to list HTML/CSS (commonly called "core") before React. We usually list core first, and that way it's clear what the parentheses refer to.